crates/ra_ide/src/syntax_highlighting/tests.rs

```diff
@@ -259,13 +259,43 @@ impl HasUnsafeFn {
     unsafe fn unsafe_method(&self) {}
 }
 
+#[repr(packed)]
+struct Packed {
+    a: u16,
+    b: u8,
```

Comment:
Do we need all three fields here? If not, lets minimize the test.

Reply:
I believe I got this from the compiler test suite (I'll double check that). The reason for the three is that in the compiler, only references to two of the fields is considered unsafe. Originally, I wanted to try supporting that, but I think (according to other comments in this PR) that won't be possible for a long time. Should this be kept with a comment or completely removed?

Reply:
Let’s remove it for now :)

Reply:
Done